### ISSUE TITLE
Aint updating

### ISSUE THREAD

**Issue by bryant-the-coder** [ZhiyuanLck/smart-pairs#24]
whenever i try to update the plugins, it shows that failed to update smart-pairs. :( 你知道怎么办吗？

**Comment by ZhiyuanLck** [comment]
Do you try to remove the plugin dir and install again?

**Comment by bryant-the-coder** [comment]
yes it worked xD. Btw u r from? Is there anyway to space out the = sign?

**Comment by bryant-the-coder** [comment]
quite interesting that you speak fluent english
